_targets.R

```diff
@@ -115,35 +117,128 @@ targets_list <- list(
 
   # Download data from GDP for each tile & GCM name combination.
   # If the cells in a tile don't change, then the tile should not need to rebuild.
   tar_target(
     gcm_data_raw_feather,
     download_gcm_data(
-      out_file_template = "7_drivers_munge/tmp/7_GCM_%s_tile%s_raw.feather",
+      out_file_template = "7_drivers_munge/tmp/7_GCM_%s_%s_tile%s_raw.feather",
       query_geom = query_cells_centroids_list_by_tile,
       gcm_name = gcm_names,
-      query_vars = gcm_vars,
-      query_dates = gcm_dates
+      gcm_projection_period = gcm_dates_df$projection_period,
+      query_vars = gcm_query_vars,
+      query_dates = c(gcm_dates_df$start_datetime, gcm_dates_df$end_datetime)
     ),
     # TODO: might need to split across variables, too. Once we scale up, our queries
     # might be too large and chunking by variable could help.
-    pattern = cross(query_cells_centroids_list_by_tile, gcm_names),
-    format = "file"
+    pattern = cross(query_cells_centroids_list_by_tile, gcm_names, gcm_dates_df),
+    format = "file",
+    error = "continue"
   ),
 
   ##### Munge GDP output into NetCDF files that will feed into GLM #####
-  # TODO - munge data
+
+  # Need to munge GCM variables into useable GLM variables
+  tar_target(glm_vars_info,
+             tibble(
+               # TODO: MISSING LONGWAVE
+               var_name = c("Rain", "AirTemp", "Snow", "RelHum", "Shortwave", "Longwave", "WindSpeed"),
+               longname = c("Total daily rainfall",
+                            "Total daily snowfall derived from precipitation and air temp",
+                            "Average daily near surface air temperature",
+                            "Average daily percent relative humidity",
+                            "Average daily surface downward shortwave flux in air",
+                            "LONGWAVE EXPLANATION",
+                            "Average daily windspeed derived from anemometric zonal and anenometric meridional wind components"),
+               units = c("meters", "meters", "degrees Celcius", "", "???", "???", "???"),
+               precision = "float"
+             )),
+  tar_target(
+    gcm_data_daily_feather,
+    munge_to_glm(gcm_data_raw_feather),
+    pattern = map(gcm_data_raw_feather),
+    format = "file"
+  ),
+
+  # Create feather files that can be used in the GLM pipeline without
+  # having to be munged and extracted via NetCDF. Temporary solution
+  # while we work out some NetCDF challenges.
+  # TODO: delete once we finish the NetCDF DSG build (see issue #252)
+  tar_target(
+    out_skipnc_feather, {
+      out_dir <- "7_drivers_munge/out_skipnc"
+      purrr::map(gcm_data_daily_feather, function(fn, gcm_names, gcm_dates_df) {
+        gcm_name <- str_extract(fn, paste(gcm_names, collapse="|"))
+        gcm_time_period <- str_extract(fn, paste(gcm_dates_df$projection_period, collapse="|"))
+
+        read_feather(fn) %>%
+          split(.$cell) %>%
```

Comment:
oh, interesting! I've never seen this `split()` function before

Reply:
It can be pretty handy! It is similar to a `group_by()` in that you specify a column to group rows of the data. Just nice here because you can do more within the next `purrr::map()` call than you would do with a `summarize()`